spec/unit/daemon_spec.rb

```diff
@@ -170,10 +170,15 @@
 
       it "should log an appropriate error message and fail miserably" do
         allow(Process).to receive(:initgroups).and_raise(Errno::EPERM)
-        error = "Operation not permitted"
-        if RUBY_PLATFORM.match("solaris2") || RUBY_PLATFORM.match("aix")
-          error = "Not owner"
-        end
+        error = case RUBY_PLATFORM
```

Comment:
This looks good, but is it possible that different/older versions of solaris will still have the older message text? Should we allow for both on solaris?